src/modules/clearCompleted.js

```diff
@@ -0,0 +1,15 @@
+const todoList = document.querySelector('.todo-list');
+
+export default function clear(list) {
+  const indexes = [];
+  list.list.forEach((item) => {
+    if (item.completed) {
+      indexes.push(item.index);
+    }
+  });
+  for (let i = indexes.length - 1; i >= 0; i -= 1) {
+    if (todoList.children[indexes[i]].children[0].children[1]) {
+      todoList.children[indexes[i]].children[0].children[1].click();
+    }
+  }
+}
```

Comment:
- [ ] The clear all completed functionality should be implemented using the Array.filter() method as stated in the requirement `Implement a function for the "Clear all completed" button (use filter() method).`